Doc/library/http.server.rst

```diff
@@ -55,6 +55,30 @@ The :class:`HTTPServer` and :class:`ThreadingHTTPServer` must be given
 a *RequestHandlerClass* on instantiation, of which this module
 provides three different variants:
 
+.. class:: HTTPSServer(server_address, RequestHandlerClass, \
+                       bind_and_activate=True, *, certfile, keyfile=None, \
+                       password=None, alpn_protocols=None)
+
+   This class is a :class:`HTTPServer` subclass with a wrapped socket using the
```

Comment:
ALPN protocols should be explained or at least mentioned. We don't what kind lf type the argument should be, or its form.